lib/rules/rule-empty-line-before/README.md

```diff
@@ -120,7 +120,7 @@ b {
 
 ## Optional secondary options
 
-### `except: ["after-rule", "after-single-line-comment", "inside-block-and-after-rule", "first-nested"]`
+### `except: ["after-rule", "after-single-line-comment", "inside-block-and-after-rule", "first-nested", "inside-block"]`
```

Comment:
Options are listed alphabetically, can you please put this before `"first-nested"`...